test_project/rename_forward/urls.py

```diff
@@ -8,6 +8,7 @@
 class LinkedDataView(autocomplete.Select2QuerySetView):
     def get_queryset(self):
         qs = super(LinkedDataView, self).get_queryset()
+        print(self.forwarded.get('checkbox', None))
```

Comment:
Yeah true, it's preferable to remove before pushing the code because then we can just merge it when someone feels like it ;)